lib/membrane_rtc_engine/endpoints/hls_endpoint.ex

```diff
@@ -25,8 +25,17 @@ if Enum.all?(
                   description: """
                   Pid of parent all notifications will be send to.
 
-                  These notifications are
+                  These notifications are:
+                    * `{:playlist_playable, content_type, stream_id}`
+                    * `{:cleanup, clean_function, stream_id}`
```

Comment:
Types for these notifications should be specified somewhere so you can reference them with `t:some_notification/0`
If theye are not let's create issue in HLS repo to fix this